rasa/core/actions/action.py

```diff
@@ -1153,6 +1154,12 @@ async def run(
         ]
 
         for slot in user_slots:
+            if (
+                tracker.active_loop
+                and tracker.active_loop.get(LOOP_REJECTED)
```

Comment:
I considered it, but then I thought this should skip the slot entirely if the active_loop is in an unhappy path and the slot is `requested_slot`. It also doesn't really have anything to do with the mapping conditions check, it would make sense though to extract this code in its own method (`_active_loop_in_unhappy_path` or something like that) 🤔